admin/acceptance.py

```diff
@@ -344,11 +350,36 @@ def generate_certificates(cluster_id, nodes):
         certificates_path,
         nodes[0].address,
         len(nodes),
+        cluster_name=cluster_name,
         cluster_id=cluster_id,
     )
     return certificates
 
 
+def make_cluster_name_and_id(purpose, dataset_backend):
+    """
+    Generate a cluster name and cluster UUID based on the cluster's purpose.
+
+    :param str purpose: The purpose of the cluster.
+                        "acceptance-testing" is a special value.
+    :param DatasetBackend dataset_backend: The dataset backend used by
+                        the cluster.
+
+    :rtype: tuple of bytes, UUID
+    :returns: The generated name and UUID of the cluster as a tuple.
+    """
+    if purpose == u'acceptance-testing':
```

Comment:
I think that this comment has become obsolete as after implementing your other suggestion the string is going to be used in only one place.